pkg/controller/controller_broker_test.go

```diff
@@ -765,6 +765,25 @@ func TestReconcileServiceBrokerFailureOnFinalRetry(t *testing.T) {
 	}
 }
 
+// TestReconcileServiceBrokerWithStatusUpdateError verifies that the reconciler
```

Comment:
I think you should indicate that this is an otherwise successful scenario where the update to set the in-progress operation fails.  Same with other similar tests.